datapackage.json

```diff
@@ -0,0 +1,217 @@
+{
+  "name": "Unicode-Codes",
+  "title": "Unicode-Codes",
+  "resources": [
+    {
+      "path": "archive/source.csv",
+      "pathType": "local",
+      "name": "source",
+      "format": "csv",
+      "mediatype": "text/csv",
+      "encoding": "ISO-8859-1",
+      "dialect": {
+        "delimiter": ",",
+        "quoteChar": "\""
+      },
+      "schema": {
+        "fields": [
+          {
+            "name": "0000",
+            "type": "integer",
+            "format": "default"
+          },
+          {
+            "name": "<control>",
+            "type": "string",
+            "format": "default"
+          },
+          {
+            "name": "Cc",
+            "type": "string",
+            "format": "default"
+          },
+          {
+            "name": "0",
+            "type": "integer",
+            "format": "default"
+          },
+          {
+            "name": "BN",
+            "type": "string",
+            "format": "default"
+          },
+          {
+            "name": "Unnamed: 5",
+            "type": "string",
+            "format": "default"
+          },
+          {
+            "name": "Unnamed: 6",
+            "type": "string",
+            "format": "default"
+          },
+          {
+            "name": "Unnamed: 7",
+            "type": "string",
+            "format": "default"
+          },
+          {
+            "name": "Unnamed: 8",
+            "type": "string",
+            "format": "default"
+          },
+          {
+            "name": "N",
+            "type": "string",
+            "format": "default"
+          },
+          {
+            "name": "NULL",
+            "type": "string",
+            "format": "default"
+          },
+          {
+            "name": "Unnamed: 11",
+            "type": "string",
+            "format": "default"
+          },
+          {
+            "name": "Unnamed: 12",
+            "type": "string",
+            "format": "default"
+          },
+          {
+            "name": "Unnamed: 13",
+            "type": "string",
+            "format": "default"
+          },
+          {
+            "name": "Unnamed: 14",
+            "type": "string",
+            "format": "default"
+          }
+        ],
+        "missingValues": [
+          ""
+        ]
+      }
+    },
+    {
+      "path": "data/data.csv",
```

Comment:
It's minor but I would name the file more descriptive and it is also a name of the resource. So `unicode.csv` or `unicode-chars.csv` is a better name.